backend/api/Areas/Notes/Controllers/NoteController.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.Generic;
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Dal;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Areas.Notes.Controllers
+{
+    /// <summary>
+    /// ContactController class, provides endpoints for interacting with contacts.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("notes")]
+    [Route("v{version:apiVersion}/[area]")]
+    [Route("[area]")]
+    public class NoteController : ControllerBase
+    {
+        #region Variables
+        private readonly IPimsRepository _pimsService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a ContactController class, initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="pimsService"></param>
+        /// <param name="mapper"></param>
+        ///
+        public NoteController(IPimsRepository pimsService, IMapper mapper)
+        {
+            _pimsService = pimsService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+
+        /// <summary>
+        /// Get the notes for the specified type.
+        /// </summary>
+        /// <param name="type">Used to identify note type.</param>
+        /// <returns></returns>
+        [HttpGet("{type}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Models.NoteModel>), 200)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [SwaggerOperation(Tags = new[] { "note" })]
+        public IActionResult GetNotes(Models.NoteType type)
+        {
+            switch (type)
+            {
+                case Models.NoteType.Activity:
+                    var notes = _pimsService.Note.GetActivityNotes();
```

Comment:
this looks like business logic so it should be handled by the service instead of the controller

Reply:
Made the changes, please have a look